### PR TITLE
My Portfolio mobile version Project 

### PR DESCRIPTION
**Pull request by Eugeneiyukhun**
In this branch, the following task was completed
- Installed linter
- fixed linter errors
- Added images and social links.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -44,8 +44,181 @@ <h2>LETS CONNECT</h2>
...
+                <h1>About Myself</h1>
+                <p>
+                    Hello I’m a software developer! I can help you build a product,
+                    feature or website Look through some of my work and experience! If
+                    you like what you see and have a project you need coded, don’t
+                    hestiate to contact me.
+                </p>
+
+                <div class="social">
+                    <p>LET’S CONNECT</p>
```
Comment (Gopxfs) [inline index.html:147]: To follow the Figma design, please change your 'LET'S CONNECT' text color to #7f8cff and give the correct spacing.
Figma design:
image

Your design:
image

File: styles.css [hunk 2]
```diff
@@ -80,9 +291,157 @@ body {
...
+  padding-top: 10px;
+  padding-bottom: 10px;
+  padding-left: 20px;
+  padding-right: 20px;
+  border-color: #6070ff;
+  margin-top: 20px;
+  background-color: white;
+  border-radius: 8px;
+  display: flex;
+}
```
Comment (Gopxfs) [inline styles.css]: To follow the Figma design, please change your 'Get My Resume' button styling and add the button effects (hover, active, etc).

Figma button:
image

Your button:
image

Figma button effects:
image

Please take into account that the objective of this project is to guide you on following design requirements. Imagine Figma as a client, who is asking you for an exact template of a website. 😉

**Review by Gopxfs — CHANGES_REQUESTED** [review]
# ♻️ STATUS: REQUIRED CHANGES ♻️

## Hi @Eugeneiyukhun 👋,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights :rocket:
✔️ Your styling is very well done!
✔️ Great code structure.
✔️ Linters are passing without errors.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 🗒️  
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Gopxfs in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Santiago220991 — APPROVED** [review]
Hi @Eugeneiyukhun,
I'm Santiago your reviewer for this PR. :wave: :wave: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
:heavy_check_mark: Followed GitHub flow.
:heavy_check_mark: Applied changes required by the last reviewer.

## Optional suggestions
[OPTIONAL] Kindly fix the typo in the LET'S CONNECT link. A good portfolio without typos makes you show more professional to the recruiters. :+1: :smiley: :smiley:  Please pay attention to the `'`.
Your design:
image
Figma design:
image

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**